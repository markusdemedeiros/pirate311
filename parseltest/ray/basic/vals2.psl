print(+ (0100, 2));
print("\n");
print(+ (100, 2));
print("\n");
print(+ (10, 11, 12));
print("\n");
print(+ (1));
"";