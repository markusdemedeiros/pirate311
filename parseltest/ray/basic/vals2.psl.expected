102
102
33
1
